File: main workshop/services/java/ride-service/src/main/java/com/powertoolsride/rideservice/model/RideCreationResult.java
package com.powertoolsride.rideservice.model;

public class RideCreationResult {
    private Ride ride;
    private boolean success;
    private String errorMessage;
    private String errorType;

    public RideCreationResult() {
        this.success = false;
        this.errorMessage = "";
        this.errorType = "";
    }

    public Ride getRide() {
        return ride;
    }

    public void setRide(Ride ride) {
        this.ride = ride;
    }

    public boolean isSuccess() {
        return success;
    }

    public void setSuccess(boolean success) {
        this.success = success;
    }

    public String getErrorMessage() {
        return errorMessage;
    }

    public void setErrorMessage(String errorMessage) {
        this.errorMessage = errorMessage;
    }

    public String getErrorType() {
        return errorType;
    }

    public void setErrorType(String errorType) {
        this.errorType = errorType;
    }
}

File: main workshop/services/dotnet/ride-service/Models/RideCreationResult.cs
using System.Text.Json.Serialization;



public class RideCreationResult
{
    [JsonPropertyName("ride")] 
    public Ride Ride { get; set; }
    
    [JsonPropertyName("success")]
    public bool Success { get; set; }
    
    [JsonPropertyName("errorMessage")]
    public string ErrorMessage { get; set; } = string.Empty;
    
    [JsonPropertyName("errorType")]
    public string ErrorType { get; set; } = string.Empty;
}